doc/user/layouts/shortcodes/session-variables.html

```diff
@@ -20,12 +20,14 @@
 emit_trace_id_notice                        | `false`                   | Boolean flag indicating whether to send a `notice` specifying the trace ID, when available.
 extra_float_digits                          | `3`                       | Adjusts the number of digits displayed for floating-point values.
 failpoints                                  |                           | Allows failpoints to be dynamically activated.
+idle_in_transaction_session_timeout         | `120 seconds`             | The maximum allowed duration that a session can sit idle in a transaction before being terminated. A value of zero disables the timeout.
```

Comment:
nit: while we're here, can you specify the units for this variable? [postgres denotes the unit as milliseconds](https://www.postgresql.org/docs/current/runtime-config-client.html), what do we do?

Reply:
Good call, we have the same behavior as Postgres. I've updated the comment for this and `statement_timeout`.